### DISCUSSION TITLE
Using Pytorch3D in Unity

### DISCUSSION THREAD

**Discussion by Ademord** [facebookresearch/pytorch3d#727] [Q&A]
Hello, I was wondering if there was ANY way i could leverage pytorch3d in Unity? A bit of context: I think this could be a massive breakthrough and i am super excited to be able to do the Fit a Mesh tutorial in unity (but later, to a set of objects which would be stored in an occupancy grid in an exploration task).

**Comment by nikhilaravi** [comment]
@Ademord We have not tried this! I am not familiar with Unity so can't comment on any specifics. PyTorch3D requires python, PyTorch and CUDA so the first step would be to figure out how to use PyTorch with Unity.

**Comment by gkioxari** [comment]
Also, this is a great discussion for Github Discussions rather than Issues. @Ademord please consider moving the discussion there.

**Comment by Ademord** [comment]
Thank you for your replies! Could you tell me how to transfer the issue to a discussion or do I just close and open there?

**Reply by krishnanpc** [reply]
@Ademord : 
Any updates on this? I am also super excited to be able to have PyTorch3D Unity.

**Reply by LeeSandberg** [reply]
Me as well.
